Nodejs/Product/Nodejs/NodejsConstants.cs

```diff
@@ -71,6 +71,9 @@ internal static bool ContainsNodeModulesOrBowerComponentsFolder(string path)
             return tmp.IndexOf("\\" + NodeModulesFolder + "\\", StringComparison.OrdinalIgnoreCase) >= 0
                 || tmp.IndexOf("\\" + BowerComponentsFolder + "\\", StringComparison.OrdinalIgnoreCase) >= 0;
         }
+
+        public const string ExecutorUriString = "executor://NodejsTestExecutor/v1";
```

Comment:
>public [](start = 8, length = 6)

Does this need to be public?

Reply:
The class is internal, so it doesn't really matter... but for aesthetic reasons i'll make them all `public`

Reply:
I was actually hoping it could be private.